_tools/lint_plugin.ts

```diff
@@ -378,5 +382,27 @@ export default {
         };
       },
     },
+    // https://docs.deno.com/runtime/contributing/style_guide/#copyright-headers
+    "copyright": {
+      create(context) {
+        // Skip checking this rule in the ignore_comments.ts file to avoid
+        // testing of other rules being affected.
+        if (context.filename === "ignore_comments.ts") return;
+        const comments = context.sourceCode.getAllComments();
+        // Only check the first 6 comments, for performance
+        const node = comments.slice(0, 6).find((comment) =>
+          comment.value.includes(COPYRIGHT_NOTICE) && comment.type === "Line"
+        );
+        if (!node) {
+          context.report({
```

Comment:
A fix was deliberately omitted as I thought it wasn't worth the added complexity to differentiate between files with missing copyright headers or out-of-date copyright headers.